src/constants/baseUrl.js

```diff
@@ -0,0 +1,6 @@
+
+export const baseUrl = "https://us-central1-missao-newton.cloudfunctions.net/rappi4"
+
+export const axiosConfig = {
+    headers: { auth: window.localStorage.getItem("token") },
+  }; 
```

Comment:
Isolar a axiosConfig dessa forma, nesse caso, não é uma boa. Nos casos em que a configuração do headers necessita de um valor do localStorage, que pode mudar com o tempo, é melhor isolar tudo em uma função e chamar a função sempre que for usar a axiosConfig, dessa forma:
```

export const generateAxiosConfig = () =>{
     const axiosConfig = {
         headers: { auth: window.localStorage.getItem("token") }
     }

    return axiosConfig
}

```